packages/mds-jurisdiction-service/service/repository/index.ts

```diff
@@ -29,7 +29,7 @@ import {
 } from '../../@types'
 import { JurisdictionEntityToDomain, JurisdictionDomainToEntityCreate } from './mappers'
 
-const isEffectiveJurisdiction = isDefined<JurisdictionDomainModel>()
+const isEffectiveJurisdiction = isDefined()
```

Comment:
Get rid of this and just use `isDefined()` instead.

Reply:
👍